client/cypress/integration/query/parameter_spec.js

```diff
@@ -170,46 +181,47 @@ describe('Parameter', () => {
         ParameterSettings-test-parameter
         ParameterTypeSelect
         DateParameterTypeOption
-        UseCurrentDateTimeCheckbox
         SaveParameterSettings
       `);
 
       const now = new Date();
       now.setDate(1);
-      cy.clock(now);
+      cy.wrap(now.getTime()).as('now');
+      cy.clock(now.getTime());
     });
 
     afterEach(() => {
       cy.clock().then(clock => clock.restore());
     });
 
-    it('updates the results after selecting a date', () => {
-      cy.getByTestId('ParameterName-test-parameter')
-        .find('input')
+    it('updates the results after selecting a date', function () {
+      selectCalendarDate('15');
+
+      cy.getByTestId('ParameterApplyButton')
         .click();
 
-      cy.get('.ant-calendar-date-panel')
-        .contains('.ant-calendar-date', '15')
+      cy.getByTestId('DynamicTable')
+        .should('contain', Cypress.moment(this.now).format('15/MM/YY'));
+    });
+
+    it('allows picking a dynamic date', function () {
+      cy.getByTestId('DynamicButton')
         .click()
-        .click(); // workaround for datepicker display bug
+        .click();
```

Comment:
Assuming the double click is the picker workaround.

```suggestion
        .click(); // workaround for datepicker display bug
```